app/core/abilities/abilities.ts

```diff
@@ -10388,12 +10389,12 @@ export class TrickOrTreatStrategy extends AbilityStrategy {
       }
     } else if (pokemon.ap <= 50) {
       // 0-50 AP: shrink unit size and HP
-      const lifeReduction = 0.4 * (1 + pokemon.ap / 100)
+      const lifeReduction = 0.4 * (1 - pokemon.ap / 100)
```

Comment:
this is incorrect, it could lead to negative life reduction. The correct formula intended is 
```suggestion
      const lifeReduction = 0.4 / (1 + pokemon.ap / 100)
```

Reply:
It's not possible regardless since it will pick a different curse above 50AP, but will do.